OMCompiler/Compiler/BackEnd/Tearing.mo

```diff
@@ -371,34 +365,38 @@ end traverseComponents1;
 protected function checkTearingSettings
 "Checks if we want to do tearing for the current component.
  It will also issue optional maesages if not."
-  input Integer maxSize;
   input Boolean isLinear;
   input Integer strongComponentIndex;
   input Integer numVars;
-  output Boolean activateTearing=false;
+  output Boolean activateTearing = false;
 protected
+  constant list<String> withLSS = {"C"} "targets that provide a linear sparse solver";
+  constant list<String> withNSS = {"C"} "targets that provide a nonlinear sparse solver";
```

Comment:
For now I've only allowed the C runtime to skip tearing in case the component is too large. Previously it was everything _except_ Cpp. @casella do you know which targets have sparse solvers and which don't?

Reply:
I'm afraid only the C runtime supports sparse solvers, thanks to Willi Braun's work circa 2015.

@rfranke can comment on cpp, and @lochel on FMI.

Reply:
The Cpp runtime generates code for sparse Jacobians and the linear solver contains Klu. It is not configured for compilation though. This is why I'd say that sparse support is experimental.
The question should not be if a target has sparse solvers. It should be if a target has sparse solvers AND if they should be used.

Reply:
I agree, maybe they could be considered for compilation also in Cpp? In fact, for phasor-based AC system simulations, sparse solvers (and daeMode!) are essential for efficient simulation.